controllers/apply/form-router-next/steps.js

```diff
@@ -98,6 +106,17 @@ module.exports = function(formId, formBuilder) {
         };
     }
 
+    async function calcLastSaveTime(applicationId) {
+        const lastUpdatedAt = (await PendingApplication.lastUpdatedTime(applicationId)).updatedAt;
+
+        return moment(lastUpdatedAt).calendar(null, {
```

Comment:
This date formatting is unusual, particularly the `L [in MM/DD/YYYY format]` line. I'd high recommend trying something like `moment(application.updatedAt).locale(locale).fromNow()` which will do all the hard work for you on formatting a relative timestamp https://momentjs.com/docs/#/displaying/fromnow/